contrib/rpmbuild/src/python/pants/contrib/rpmbuild/targets/remote_source.py

```diff
@@ -0,0 +1,63 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.base.payload import Payload
+from pants.base.payload_field import PrimitiveField
+from pants.build_graph.resources import Resources
+
+
+class RemoteSource(Resources):
```

Comment:
So unbeknownst to me, Pants already has a RemoteSources target. It is for something completely different, unpacking a jar file. But this concept will probably have to be renamed in order to not cloud the API.

RemoteFile?

Reply:
Should the target type have "rpm" in the name? (This would isolate it to the rpmbuild plugin until such time as the concept is accepted into core Pants.)

So maybe `rpm_remote_file`?